obspy/core/trace.py

```diff
@@ -562,7 +562,7 @@ def __div__(self, num):
             tend = tstart + (self.stats.delta * packet_length)
         return st
 
-    # Py3k: '/' does not map to __div__ anymore in Python 3
+    # Py3k: '/' does not map to __truediv__ anymore in Python 3
     __truediv__ = __div__
```

Comment:
I don't think this is correct; you probably want to rename the method to `__truediv__` and remove `__div__`.

Reply:
Not sure. I didn't even know that this functionality existed.
In the tests it is used in the following way

```
        tr = Trace(data=np.arange(1000))
        st = tr / 5
        assert len(st) == 5
```
but I agree that using only `tr // 5` would make sense.

Reply:
That one is `__floordiv__`; there is no `__div__` any more.

Reply:
Thanks, now I understand.